ts/src/main/scala/ch/epfl/ts/indicators/OhlcIndicator.scala

```diff
@@ -1,14 +1,14 @@
 package ch.epfl.ts.indicators
 
 import ch.epfl.ts.component.Component
-import ch.epfl.ts.data.{OHLC, Transaction, Quote}
-
+import ch.epfl.ts.data.{ OHLC, Transaction, Quote }
 import scala.collection.mutable.MutableList
+import ch.epfl.ts.data.Currency
 
 /**
  * computes OHLC tick for a tick frame of the provided size, the OHLCs are identified with the provided marketId
  */
-class OhlcIndicator(marketId: Long, tickSizeMillis: Long) extends Component {
+class OhlcIndicator(marketId: Long, symbol: String, tickSizeMillis: Long) extends Component {
```

Comment:
The `Currency` type is available to the user and clearly defines which currencies we support. We should strive to use it wherever possible for safety and consistency.